core/arch/arm/plat-stm32mp1/conf.mk

```diff
@@ -35,9 +35,15 @@ CFG_WITH_LPAE ?= y
 CFG_WITH_STACK_CANARIES ?= y
 CFG_MMAP_REGIONS ?= 23
 
+ifeq ($(CFG_EMBED_DTB_SOURCE_FILE),)
+# Some drivers mandate DT support
```

Comment:
Is it DT who require I2C or I2C who requires DT? I get puzzled when reading the comment and seeing the code.

Reply:
I2C requires DT. Main concern is I2C configuration that mandate quite a lot of settings which we to decide to support only using DT means. If embedded DT is disabled, I2C driver is forced disabled.